aten/src/ATen/FunctionalTensorWrapper.h

```diff
@@ -165,6 +165,10 @@ struct TORCH_API FunctionalTensorWrapper : public c10::TensorImpl {
   // See Note[resize_() in functionalization pass]
   void maybe_replace_storage(const Tensor& other);
 
+  // Replaces the storage with a new functional storage,
+  // and clears the view_metas_ stack.
+  void reset_storage();
```

Comment:
Can you rename this to `_unsafe_reset_storage`, and mention that it will sever the aliasing relationship between the current `FunctionalTensorWrapper` and any of its outstanding aliases?